load.php

```diff
@@ -41,6 +41,12 @@ function bootstrap( $wp_debug_enabled ) {
 		die( 'HM Platform is only supported on WordPress 4.6+.' );
 	}
 
+	// Disable indexing when not in production
+	// Original source: https://github.com/roots/bedrock/blob/master/web/app/mu-plugins/disallow-indexing.php
```

Comment:
Let's drop the source, it's so simple it doesn't need attribution I think. Can we also have a way to disable this behavior? A filter, or something - as we have some sites where we wouldn't want this.